src/main/java/org/mdbenefits/app/cli/GoogleDriveClient.java

```diff
@@ -39,12 +39,23 @@ public interface GoogleDriveClient {
     List<File> findDirectory(String name, String parentId);
 
     /**
-     * Deletes the specified directory with the matching directory ID.
+     * Hard deletes the specified directory with the matching directory ID; this bypasses the trash feature. Only "Managers" in a
+     * Google Drive team environment can use this call. A "Content Manager" will not have sufficient permissions to use this.
      *
```

Comment:
non-blocking: Consider adding a "see also" that references the `trashDirectory` method.